src/api/uploads/BaseUpload.js

```diff
@@ -5,9 +5,10 @@
  */
 
 import Base from '../Base';
-import { DEFAULT_RETRY_DELAY_MS, MS_IN_S } from '../../constants';
+import { DEFAULT_RETRY_DELAY_MS, MS_IN_S, DEFAULT_HOSTNAME_UPLOAD } from '../../constants';
 
 const MAX_RETRY = 5;
+const MAX_REACHABILITY_RETRY = 10;
```

Comment:
I don't understand how we landed on 10 as max retries. Do we know the max possible different upload hosts that could come back? I'd imagine that would determine the max reachability retries before we should default to the default upload host.

Reply:
Copied the value from what the current max retries in php, which is 10. From this looks like we currently have 7 different upload hosts https://support.box.com/hc/en-us/articles/360043694474-Box-Connectivity-Tests.